pkg/skaffold/build/kaniko/sources/sources.go

```diff
@@ -81,7 +81,39 @@ func podTemplate(cfg *latest.KanikoBuild, args []string) *v1.Pod {
 						SecretName: cfg.PullSecretName,
 					},
 				},
-			}},
+			},
+			},
 		},
 	}
+
+	if cfg.MountDockerConfig {
```

Comment:
It might be cleaner here to check the negative, so that a bunch of code isn't nested in the if statement:

```golang
if !cfg.MountDockerConfig {
  return pod
}
```

Reply:
Done